### DISCUSSION TITLE
Status Change Notification

### DISCUSSION THREAD

**Discussion by ingdesarr1** [FreeOpcUa/python-opcua#1537] [Q&A]
Good afternoon. I received the next StatusChangeNotification, but I dont know what it means. Can somebody help me? thanks

StatusChangeNotification(Status=StatusCode(value=2148270080), DiagnosticInfo_=DiagnosticInfo(SymbolicId=None, NamespaceURI=None, Locale=None, LocalizedText=None, AdditionalInfo=None, InnerStatusCode=None, InnerDiagnosticInfo=None))

This message is received when the PLC reset or recharge the program.

**Comment by harriv** [comment]
It seems to be "BadShutdown". 

"The operation was cancelled because the application is shutting down"

https://www.opcti.com/common-error-codes.aspx


![image](https://github.com/FreeOpcUa/python-opcua/assets/2283563/dc4983f0-5393-45a1-9dec-e31fd5d4562a)
